src/protobuf/io/bazel/rules_scala/diagnostics.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
```

Comment:
In the long term, this should probably live in the Bazel repo rather than the `rules_scala` one, but it's probably fine here for now for getting something going :)

Reply:
Perhaps it would be an interesting idea to already place it there when implementing the javac diagnostics

Reply:
indeed sounds less rules_scala oriented then general infra of bazel bsp.
I can live with it being here for now

Reply:
to be clear- I didn't thoroughly review this since I assume this is belongs to your domain and is here as an evolutionary step

Reply:
I don't think this file will be here by the end of this PR, since the implementation of Diagnostics in Java required this to live inside the `bazel` repository